src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/MediationSuccessfulApplicantNotificationHandler.java

```diff
@@ -45,15 +52,59 @@ public String camundaActivityId(CallbackParams callbackParams) {
 
     private CallbackResponse notifyApplicant(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
-        notificationService.sendMail(
-            caseData.getApplicantSolicitor1UserDetails().getEmail(),
-            notificationsProperties.getNotifyApplicantLRMediationSuccessfulTemplate(),
-            addProperties(caseData),
-            String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference()));
+        Boolean isCarmEnabled = featureToggleService.isCarmEnabledForCase(caseData);
+        if (isCarmEnabled) {
+            String claimId = caseData.getCcdCaseReference().toString();
+            String referenceTemplate = String.format(REFERENCE_TEMPLATE, claimId);
+            MultiPartyScenario scenario = getMultiPartyScenario(caseData);
+            // LIP
+            if (caseData.isLipvLipOneVOne()) {
+                sendEmail(
+                    caseData.getApplicant1().getPartyEmail(),
+                    notificationsProperties.getNotifyLipClaimantSuccessfulMediation(),
+                    lipClaimantProperties(caseData),
+                    referenceTemplate
+                );
+            } else
+                // 1V2 -> same and different defendants
+                if (scenario.equals(ONE_V_TWO_ONE_LEGAL_REP) || scenario.equals(ONE_V_TWO_TWO_LEGAL_REP)) {
```

Comment:
can the `else if` be on the same line and indented accordingly?
this makes it harder to read because it looks like 

if { } else { if{} else{} }
when it is actually >> if {} else if {} else {}

Just for easier readability.

so for context
```
if (caseData.isLipvLipOneVOne()) {
                sendEmail(
                    caseData.getApplicant1().getPartyEmail(),
                    notificationsProperties.getNotifyLipClaimantSuccessfulMediation(),
                    lipClaimantProperties(caseData),
                    referenceTemplate
                );
} else if (scenario.equals(ONE_V_TWO_ONE_LEGAL_REP) || scenario.equals(ONE_V_TWO_TWO_LEGAL_REP)) {
     // 1V2 -> same and different defendants
     //send notification to the defendant
                    sendEmail(
                        caseData.getApplicantSolicitor1UserDetails().getEmail(),
                        notificationsProperties.getNotifyOneVTwoClaimantSuccessfulMediation(),
                        oneVtwoProperties(caseData),
                        referenceTemplate
                    );
} else {
                    // LR scenarios
                    sendEmail(
                        caseData.getApplicantSolicitor1UserDetails().getEmail(),
                        notificationsProperties.getNotifyLrClaimantSuccessfulMediation(),
                        lrClaimantProperties(caseData),
                        referenceTemplate
                    );
}
```

Reply:
fixed.